cli/gumball-machine/README.md

```diff
@@ -0,0 +1,184 @@
+# Gumball Machine CLI
+
+## Getting Started
+
+See README.md in the parent directory.
+
+## Usage
+
+This is a CLI for Gumball Machine which generally sits on top of the TypeScript SDK in `candyland/contracts/sdk/gumball-machine`. It provides CLI commands for each core instruction that Gumball Machine supports. Note that before using the CLI, it's critical that you have access to an RPC URL for a network which has the entire Candyland suite deployed (SugarShack not required).
+
+### Key Design Choice
+
+In general, each command corresponds to an instruction in the Gumball Machine smart contract. All accounts that an instruction requires are generally specified as required options for the corresponding command. For more information about the options/arguements of each command you can run `ts-node gumball-machine-cli.ts <command_name> help`. 
+
+However, in general, the *arguements* to each instruction are specified in the template file associated with each command. See `/templates` for more. To be clear, a user should modify these template files each time they wish to change the arguements they are passing to the associated on-chain instruction.
```

Comment:
Can we set this up so user can provide their own config (you choose: *js, *ts, or JSON) instead of hardcoding path to `template/<ix>.ts`? I like this design decision.

also typo: *arguments